src/services/config/index.ts

```diff
@@ -66,7 +66,7 @@ class ConfigService {
 
   public userConfigMustBeVaild = (userId: string): boolean => {
     const userConfig = this.store.getAllUserConfigs()[userId]
-    if (!userConfig?.analyticsOptIn || !userConfig?.activeProjectId) {
+    if (!userConfig?.analyticsOptIn === undefined || !userConfig?.activeProjectId) {
```

Comment:
Refactor this repetitive check into a validation function that could abstract away the logic